app/core/BackgroundBridge/BackgroundBridge.js

```diff
@@ -100,8 +100,11 @@ export class BackgroundBridge extends EventEmitter {
     );
     Engine.context.PreferencesController.subscribe(this.sendStateUpdate);
 
-    Engine.context.KeyringController.onLock(this.onLock.bind(this));
-    Engine.context.KeyringController.onUnlock(this.onUnlock.bind(this));
+    Engine.controllerMessenger.subscribe('KeyringController:lock', this.onLock);
```

Comment:
We probably want to still bind this:

```suggestion
    Engine.controllerMessenger.subscribe('KeyringController:lock', this.onLock.bind(this));
```

Or wrap it in an arrow function:

```suggestion
    Engine.controllerMessenger.subscribe('KeyringController:lock', () => this.onLock());
```

Otherwise `this` will not be set in the `onLock` function. The same goes for the other handlers as well.